packages/amesos2/src/Amesos2_TpetraMultiVecAdapter_def.hpp

```diff
@@ -335,7 +326,7 @@ namespace Amesos2 {
       // num_vecs = 1; stride does not matter
       auto mv_view_to_modify_2d = mv_->template getLocalView<host_execution_space>();
       for ( size_t i = 0; i < lda; ++i ) {
-        mv_view_to_modify_2d(i,0) = new_data[i];
+        mv_view_to_modify_2d(i,0) = new_data[i]; // Only one vector
```

Comment:
Why not do a `deep_copy` instead of writing a for loop?

Reply:
@mhoemmen thanks for feedback! This was code I mostly worked on during my first go at using Tpetra awhile back, I wasn't sure if `deep_copy` from an `ArrayView` (`new_data`) and a 1d View was supported - I'm supposing it must be based on the suggestion.

Reply:
(Just to clarify -- the only way to use `deep_copy` with `ArrayView` right now is to turn the `ArrayView` into an unowned `HostSpace` `View`.  You did that above :) .)

Reply:
You'll also want to fix this bit in the same way that you fixed it above.  Also, if you know that the MultiVector has only one column, you could turn that 2-D View into a 1-D View first, before doing the copy.